package.json

```diff
@@ -6,6 +6,7 @@
     "dev": "turbo dev",
     "start": "turbo start",
     "build": "turbo build",
+    "build:exclude-web": "turbo build --filter=!@daimo/web",
```

Comment:
👍

Reply:
where is this used?

Reply:
i think he just wanted a much faster alternative to `npm run build` haha